### PR TITLE
feat(en-316): implement breadcrubms for the enterprise demo

### PR DESCRIPTION
**Pull request by lsliwaradioluz**
### Related Issues

closes [EN-316](https://vsf.atlassian.net/browse/EN-316)

### Short Description of the PR

I've replaced the hard-coded breadcrumbs on the PDP with an Array of agnostic breadcrumbs returned by a brand-new `productGetters.getBreadcrumbs` method. 

The feature also required to update the `getProduct`'s default query (to fetch categories along with categoriesRef) and adjusting the `enhanceProduct` method and make it preserve the `categories` array as well. 

This implementation - as requested in [EN-316](https://vsf.atlassian.net/browse/EN-316) - assumes that we want only the topmost category of the product to be included in breadcrumbs. Hence it now has the shape:

`Home` - `Topmost product category` - `Product name`

### Screenshots of Visual Changes before/after (if There Are Any)

image

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

File: packages/commercetools/composables/src/getters/productGetters.ts [hunk 1]
```diff
@@ -112,7 +124,8 @@ const productGetters: ProductGetters<ProductVariant, ProductVariantFilters> = {
   getSku: getProductSku,
   getFormattedPrice,
   getTotalReviews,
-  getAverageRating
+  getAverageRating,
+  getBreadcrumbs: getProductBreadcrumbs
```
Comment (dawid-ziobro) [inline packages/commercetools/composables/src/getters/productGetters.ts:128]: It should also be added in boilerplate

Comment (lsliwaradioluz) [reply packages/commercetools/composables/src/getters/productGetters.ts:128]: ✔️ added [here](https://github.com/vuestorefront/vue-storefront/pull/6111/commits/f266a63fd7801a34d75d010e654517faf631916f)

Comment (dawid-ziobro) [reply packages/commercetools/composables/src/getters/productGetters.ts:128]: Thanks!

**Comment by lsliwaradioluz** [comment]
The feature has been implemented in our new CT repository.

**Pull request closed without merging**